disperser/batcher/metrics.go

```diff
@@ -249,3 +250,42 @@ func (t *TxnManagerMetrics) UpdateTxQueue(txQueue int) {
 func (t *TxnManagerMetrics) IncrementTxnCount(state string) {
 	t.NumTx.WithLabelValues(state).Inc()
 }
+
+type DynamoDBCollector struct {
+	blobMetadataStore *blobstore.BlobMetadataStore
+	blobStatusMetric  *prometheus.Desc
+	logger            common.Logger
+}
+
+func NewDynamoDBCollector(blobMetadataStore *blobstore.BlobMetadataStore, logger common.Logger) *DynamoDBCollector {
+	return &DynamoDBCollector{
+		blobMetadataStore: blobMetadataStore,
+		blobStatusMetric: prometheus.NewDesc("dynamodb_blob_metadata_status_count",
+			"Number of blobs with specific status in DynamoDB",
+			[]string{"status"},
+			nil,
+		),
+		logger: logger,
+	}
+}
+
+func (collector *DynamoDBCollector) Describe(ch chan<- *prometheus.Desc) {
+	ch <- collector.blobStatusMetric
+}
+
+func (collector *DynamoDBCollector) Collect(ch chan<- prometheus.Metric) {
+	metadatas, err := collector.blobMetadataStore.GetBlobMetadataByStatus(context.Background(), disperser.Processing)
```

Comment:
We should be adding a new method in blob metadata store that just returns count for given status. 
This method downloads all metadata for given status and will truncate to first 1MB worth of data if there is more.